act-rules-format.md

```diff
@@ -206,6 +215,82 @@ For each accessibility requirement in the mapping, an ACT Rule <em class="rfc211
   </ul></blockquote>
 </aside>
 
+#### Secondary Requirements ####
+
+When both the "Failed Outcomes" and "Passed or Inapplicable Outcomes" described under [=Conformance Requirements=] are true for an accessibility requirement, the rule <em class="rfc2119">must</em> map that accessibility requirement as a Conformance Requirement.
+
+When only one of the Outcomes is true or both Outcomes are only sometimes true for an accessibility requirement, the rule <em class="rfc2119">should</em> map the accessibility requirement as Secondary. When an ACT rule maps to a Secondary requirement, it must include an explanation of why that requirement is Secondary in the Background section of the rule.
+
+Some scenarios when a rule will have Secondary requirements include:
+
+**Scenario 1**: only the "Failed Outcomes" are true for the requirement
+
+A rule was designed to test a minimum accessibility requirement and there exists a stricter requirement. In this scenario, the rule’s failed outcomes can determine that the stricter accessibility requirement is not satisfied, and the rule’s passed outcomes may not determine that the stricter requirement is satisfied. It is possible that the accessibility requirement may be not satisfied when the rules outcomes are passed. The stricter requirement is a Secondary requirement.
+
+<aside class=example>
+  <header>Example: a rule that tests if text has minimum contrast</header>
+<blockquote>This rule was designed to test Success Criterion 1.4.3 Contrast Minimum (AA). A stricter requirement, Success Criterion 1.4.6 Contrast (Enhanced) (Level AAA), will be not satisfied when the rule outcome is failed, and may be not satisfied when the rule outcomes are passed. This rule’s mapping:
```

Comment:
I'm afraid that doesn't match the definition:
> - Passed or Inapplicable Outcomes: When all of the outcomes are `passed` or `inapplicable`, the accessibility requirement could be <dfn>satisfied</dfn> or <dfn>further testing is needed</dfn>.

Now, when the rule passes, both 1.4.3 and 1.4.6 may be not satisfied on the page. For 1.4.3 because it also applies to images of text and the rule doesn't; for 1.4.6 because of images of text and the higher threshold. So this condition is true for both (and so is the "Failed Outcomes" condition).

I definitely think the idea is good but I don't think the definition is good yet 😕 
I think the difference we need to make is between "all outcomes are `passed` => the requirement may be satisfied or not satisfied **for the test subject** (=classical further testing is needed)" versus "an outcome is `passed` => the requirement may be satisfied or not satisfied **for this test target**".

The first one describes the relation of the rule for both 1.4.3 and 1.4.6, but the second only describes it for 1.4.6.
I could not find a way to describe the relation we want for 1.4.3 without breaking elsewhere (e.g. "an outcome is `passed` => the requirement is satisfied for this test target" wouldn't describe the mapping of "image has accessible name" to 1.1.1.

Anyway, I feel that with the current condition on Passed outcome, 1.4.6 is still a conformance mapping for such a rule, which is not what we want.

Reply:
@Jym77  - there have been updates since you left this and the other comments. Do the latest changes address your concerns?

Reply:
But for this case (out of the 4 scenarios), when the rule fails, 1.4.6 is not satisfied.
So the difference between 1.4.3 and 1.4.6 on this rule is solely on the intention of the rule author.

> When an ACT Rule is designed to test conformance to one or more [=Accessibility requirements documents=], the rule <em class="rfc2119">must</em> list all [=accessibility requirements=] from those documents that are not satisfied when one or more of the [=outcomes=] of the rule is `failed`.

This states that 1.4.6 **must** be listed as a requirement (either conformance or secondary).

> When a rule **is designed** to test a specific accessibility requirement, the accessibility requirement is mapped as a Conformance Requirement when both of the following conditions are true:

(emphasise mine)
This state that 1.4.6 is not necessarily a "conformance requirement" because the rule "is designed" to test 1.4.3.

So, in this case, the only difference is the author intention. (for the other 3 scenarios, it is also the mapping, I agree).
There is the suggestion I made and the note that has been added that we  could use strictness to make sure that in this case 1.4.6 can only be a secondary requirement. I would prefer to have it normatively since it would avoid the discussions whether 1.4.6 should be "conformance" or "secondary" in this case. But it does work as it is now, and maybe it is good enough to leave it like that rather than add complexity…